doc/release/release_0.14.rst

```diff
@@ -0,0 +1,273 @@
+Announcement: scikit-image 0.14.0
+=================================
+
+We're happy to announce the release of scikit-image v0.14.0!
+
+scikit-image is an image processing toolbox for SciPy that includes algorithms
+for segmentation, geometric transformations, color space manipulation,
+analysis, filtering, morphology, feature detection, and more.
+
+This is the last major release with official support for Python 2.7. Future
```

Comment:
IMHO I prefer `Python 2.7; future` because it signals more of a connection between the statements while still being grammatically correct.